mesh/pmesh.cpp

```diff
@@ -835,12 +837,26 @@ ParMesh::ParMesh(const ParNCMesh &pncmesh)
    , MyRank(pncmesh.MyRank)
    , gtopo(MyComm)
    , pncmesh(NULL)
+   , have_global_element_offset(false)
 {
    Mesh::InitFromNCMesh(pncmesh);
    ReduceMeshGen();
    have_face_nbr_data = false;
 }
 
+void ParMesh::ComputeGlobalElementOffset() const
+{
+   long local_elems = NumOfElements;
+   long total_elems = 0;
+   MPI_Allreduce(&local_elems, &total_elems, 1, MPI_LONG, MPI_SUM, MyComm);
```

Comment:
Is this necessary? Looks like `total_elems` is not used...

Reply:
I think you're right.  Will remove.